src/libs/antares/study/parameters.cpp

```diff
@@ -1633,7 +1661,15 @@ void Parameters::saveToINI(IniFile& ini) const
         // Unfeasible problem behavior
         section->add("include-unfeasible-problem-behavior",
                      Enum::toString(include.unfeasibleProblemBehavior));
-        section->add("solver-logs", solverLogs);
+        section->add("solver-logs", optOptions.solverLogs);
+        if (optOptions.ortoolsSolver == "xpress")
+        {
+            section->add("xpress-parameters", optOptions.solverParameters);
+        }
+        else if (optOptions.ortoolsSolver == "scip")
+        {
+            section->add("scip-parameters", optOptions.solverParameters);
+        }
```

Comment:
This is very risky. You should have one field for each solver, to avoid losing information when changing the solver.

```suggestion
            section->add("xpress-parameters", optOptions.solverParameters.xpress);
            section->add("scip-parameters", optOptions.solverParameters.scip);
        }
```